static/js/github-edit-button.js

```diff
@@ -0,0 +1,32 @@
+(function addGithubEditButton() {
+    $('#edit-on-github').click(function (e) {
+        if (location.hostname.indexOf("localhost") != -1 ) 
+        {
+            alert("Disabled for localhost")
+            return;
+        }
+        var path = location.pathname;
+        if (path.indexOf("/", path.length - 1) !== -1) path = path.slice(0, -1);
+        if (path.indexOf(".html") == -1) path += "/index.html";
+        var branch = getBranch();
+        location.href = getGithubUrl(branch);;
+    });
+})()
+
+function getBranch() 
+{
+    var branch = location.pathname.split("/")[1];
+    if (branch == "dev") 
+        branch = "master";
+    version = new RegExp("\\d.\\d.\\d");
```

Comment:
i'd rename it versionPattern so it hints at being a regexp and not a value
you can also shorten regexp to ```/(\d\.){2}\d/``` instead of ```new RegExp(...)```